### DISCUSSION TITLE
Which camera is this and can openmiko used for it?

### DISCUSSION THREAD

**Discussion by GCV-Sleeper-Service** [openmiko/openmiko#139] [General]
Hello everyone,

Back >4 years ago I got this Dafang 1s camera (as it was in the description) from Aliexpress. Visually it looks like identically as Wyze cam 2. It was sitting in the box for 4 years and I finally decided to put it into action. However I am not sure if the firmware can be applied to it.

This is the sticker on the camera itself
![image](https://github.com/user-attachments/assets/f9121da0-94f0-4005-a390-6c428a0b5a9d)

And this is the sticker/text on the camera box itself:
![image](https://github.com/user-attachments/assets/e3f96a81-1ab0-48a8-ad67-7e45e52a8afb)

If you can help me to determine what this camera is and if it is compatible with openmiko - I would greately appriciate it.

**Comment by martinbutt** [comment]
The label looks like the one on a Dafang 1S, so yes, Openmiko will work with it. Please note that the latest stable build is [v0.0.40](https://github.com/openmiko/openmiko/releases/tag/v0.0.40).
